demos/mlsys_2020/demo_source.py

```diff
@@ -3,10 +3,13 @@
 import drawSvg as draw
 from drawSvg.widgets import DrawingWidget, AsyncAnimation
 import ipywidgets as widgets
+from IPython.display import display
+from ipywidgets import Layout
 
 import plaidml
 import plaidml.exec
 from plaidml.edsl import *
+from ipywidgets import Textarea, VBox
```

Comment:
Can you sort your imports? If you're using vscode, there's a command to do that for you.